website/templates/project/register.mako

```diff
@@ -6,7 +6,7 @@
 <%def name="title()">Registration of ${node['title']}</%def>
   <div id="registrationMetaDataScope" class="container scripted">
     <div class="row">
-      <div class="span8 col-md-2 columns eight large-8">        
+      <div class="span8 col-md-2 columns eight large-8">
```

Comment:
Ah, right- the original list of styles got buried on a different PR. It's hard to keep track of who saw what comment- sorry.

Basically, this templatate references a mix of bootstrap 2 and 3 styles (`span8` is deprecated`), plus some that appear to be derived from other libraries entirely and aren't used in the page (`eight`, `large8`, `columns`).

There may be some trial and error to clean up the templates here, but that should give a start.